api/README.md

```diff
@@ -20,3 +21,104 @@ Specifically, `uint64` types aren't strictly supported by OpenAPI. So we added a
 ## Why do we have indexer.oas2.yml and indexer.oas3.yml?
 
 We chose to maintain V2 and V3 versions of the spec because OpenAPI v3 doesn't seem to be widely supported. Some tools worked better with V3 and others with V2, so having both available has been useful. To reduce developer burdon, the v2 specfile is automatically converted v3 using [converter.swagger.io](http://converter.swagger.io/).
+
+# Fixtures Test
+## What is a **Fixtures Test**?
+
+Currently (September 2022) [fixtures_test.go](./fixtures_test.go) is a library that allows testing Indexer's router to verify that endpoints accept parameters and respond as expected, and guard against future regressions. [app_boxes_fixtures_test.go](./app_boxes_fixtures_test.go) is an example _fixtures test_ and is the _creator_ of the fixture [boxes.json](./test_resources/boxes.json).
+
+A fixtures test
+
+1. is defined by a go-slice called a _Seed Fixture_  e.g. [var boxSeedFixture](https://github.com/algorand/indexer/blob/b5025ad640fabac0d778b4cac60d558a698ed560/api/app_boxes_fixtures_test.go#L302-L692) which contains request information for making HTTP requests against an Indexer server
+2. iterates through the slice, making each of the defined requests and generating a _Live Fixture_
+3. reads a _Saved Fixture_ from a json file e.g. [boxes.json](./test_resources/boxes.json)
+4. persists the _Live Fixture_ to a json file not in source control
+5. asserts that the _Saved Fixture_ is equal to the _Live Fixture_
+
+In reality, because we always want to save the _Live Fixture_ before making asserstions that could fail the test and preempt saving, steps (3) and (4) happen in the opposite order.
```

Comment:
```suggestion
In reality, because we always want to save the _Live Fixture_ before making assertions that could fail the test and pre-empt saving, steps (3) and (4) happen in the opposite order.
```